kafkabp/config_test.go

```diff
@@ -108,4 +109,32 @@ func TestConsumerConfig(t *testing.T) {
 			t.Errorf("expected sarama rack id to be empty, got %q", sc.ClientID)
 		}
 	})
+
+	cfg.AutoCommitIntervalSeconds = 8
+	t.Run("auto-commit-interval", func(t *testing.T) {
+		sc, err := cfg.NewSaramaConfig()
+		if err != nil {
+			t.Errorf("expected no error, got %v", err)
+		}
+		if sc == nil {
+			t.Fatal("expected config to be non-nil, got nil")
+		}
```

Comment:
could probably simplify this:

```suggestion
		sc, err := cfg.NewSaramaConfig()
		if err != nil {
			t.Fatalf("expected no error parsing config, got %v", err)
		}
```

You can assume (it's idiomatic Go) if no error is returned that the other return value is non-nil.